sync-todo/v2/client/kotlin-sdk/README.md

```diff
@@ -4,8 +4,8 @@
 
 Ensure `app/src/main/res/values/atlasConfig.xml` exists and contains the following properties:
 
-- **realm_app_id:** your Atlas App Services App ID.
-- **realm_base_url** the App Services backend URL. This should be https://realm.mongodb.com in most cases.
+- **app_id:** your Atlas App Services App ID.
```

Comment:
Ack. Sorry. This is set by the backend and can't be changed. Forgot to clarify.

Reply:
np! Reverted back.